resources/dicts/thoughts/alive/general.json

```diff
@@ -2100,14 +2100,14 @@
             "Had a tearful visit from r_c in {PRONOUN/m_c/poss} dream last night",
             "Wants to work hard knowing r_c is watching {PRONOUN/m_c/object}"
         ],
-        "main_living_status": [
+        "random_living_status": [
             "starclan",
             "darkforest"
         ],
         "relationship_constraint": [
             "child/parent"
         ],
-        "random_living_status": [
+        "main_living_status": [
```

Comment:
this thought group is in thoughts/alive, so this is superfluous, removing it would make for cleaner code